src/cljc/ataru/schema/form_schema.cljc

```diff
@@ -133,13 +133,15 @@
    :key                             s/Str
    :lang                            s/Str
    :state                           s/Str
+   (s/optional-key :secret)         s/Str
```

Comment:
Does this need to be an optional-key, since we're migrating all the old stuff and new applications also should always have the secret set?